README.md

```diff
@@ -1 +1,87 @@
-# functional-harness
+## testcafe-functional-harness
+
+Runs functional tests in local browsers and on SauceLabs.
+
+#### What it does
+
+1) Starts the server with the test pages.
+2) Opens the specified browsers (locally or on SauceLabs).
+3) Runs mocha tests.
+4) Closes the browsers and stops the server after all tests are finished.
+
+#### Install:
+
+```
+$ npm install testcafe-functional-harness
+```
+#### How to use:
+
+##### run-tests.js
```

Comment:
I suppose we should call it test-runner.js and it should export:
`run()`
`testCafe`
`browserConnections`

Because of the following code looks a little strange:
```js
var runTests = require('run-tests');

var tc = runTests.testCafe;    // runTests looks like a function but in fact has some other fields
```